source/PListItem.h

```diff
@@ -5,6 +5,7 @@
 #include <View.h>
 #include <Entry.h>
 #include <stdlib.h>
+#include <cstring>
```

Comment:
PListItem.h doesn't need this header.

Reply:
Yes it does, for strcmp.

Reply:
Do you see any strcmp in https://github.com/HaikuArchives/Peek/blob/master/source/PListItem.h?

Reply:
![complaints](https://user-images.githubusercontent.com/57174311/72273624-3ce94380-365d-11ea-98ce-53ac009f5be2.png)
Here are `make`'s complaints when I comment out the `<cstring>` header.

Reply:
These complaints are in PListItem.cpp

Reply:
Oh, I see it now. I've moved the header include to the `PListItem.cpp` instead. 
The changes are added to my other pull request, #20